### DISCUSSION TITLE
ESP32+W5500 shield   + WebSockets2_Generic  example

### DISCUSSION THREAD

**Discussion by dirkdickler** [khoih-prog/WebSockets2_Generic#51] [Ideas]
Hello is possible to add example code to example folder for this configuration?  
I found only  ESP32 with WiFi  :-(

**Comment by khoih-prog** [comment]
Hi @dirkdickler 

The new [**WebSockets2_Generic** v1.11.0](https://github.com/khoih-prog/WebSockets2_Generic/releases/tag/v1.11.0) has just been published to add support to **ESP32 using W5x00 Ethernet**

Check the new examples in [**ESP32 + W5x00**](https://github.com/khoih-prog/WebSockets2_Generic/tree/master/examples/Generic/Ethernet/ESP32)

---

### Release v1.11.0

1. Add support to `ESP32` using `W5x00` Ethernet
2. Update `Packages' Patches`

**Reply by dirkdickler** [reply]
Hello, thanks for support- I'm on way but on monday I will test it,  I bought 5 coffies for this excelent support.

**Comment by khoih-prog** [comment]
The new [**WebSockets2_Generic** v1.12.0](https://github.com/khoih-prog/WebSockets2_Generic/releases/tag/v1.12.0) has just been published to add support to **ESP32 using ENC28J60 with either EthernetENC or UIPEthernet library**

You can use the same examples for ESP32 in [**ESP32 Ethernet**](https://github.com/khoih-prog/WebSockets2_Generic/tree/master/examples/Generic/Ethernet/ESP32)


---

### Release v1.12.0

1. Add support to **ENC28J60** using `EthernetENC` or `UIPEthernet` Library for these supported boards

- ESP32
- nRF52
- RP2040
- SAM DUE
- SAMD
- STM32
- Teensy
